css/style.css

```diff
@@ -150,15 +164,17 @@ header #burger-button:hover + nav {
   top: 0;
   background-color: #6070ff;
   mix-blend-mode: multiply;
+  z-index: 3;
 }
```

Comment:
## Recommendation Change
This change will help improve the way your app looks. It's not a significant change to ask for a fix and resubmit for review but I trust you can fix it.

- [ ] To complete the blurred background for the mobile menu background, kindly add this opacity style below these styles. `opacity: .8;`. What do you think? 👍

Reply:
I think you are a genius I did that thanks for the suggestion